lib/dfe_sign_in_api.rb

```diff
@@ -4,22 +4,22 @@ class ForbiddenRequestError < StandardError; end
   class UnknownResponseError < StandardError; end
 
   class API
-    PAGE_SIZE = 1000
+    PAGE_SIZE = 25
 
     def users(page: 1)
       perform_request('/users', page)
     end
 
     def approvers(page: 1)
-      perform_request('/users/approvers', page)
+      perform_request('/users/approvers', page, 1000)
```

Comment:
This number could be a constant too, like `APPROVERS_PAGE_SIZE`

Reply:
🤔

>  Requesting 1000 records per page results in External Server error, change this back to the default 25 records

Either your message is wrong, or this diff is wrong.

Reply:
@adrianclay This is the `approver` endpoint which works fine with 1000 request, but the `users` endpoint (line 10) doesn't. I realised the message wasn't clear enough, so I updated it! 

@cesidio I was wondering if there is any value making this a constant since we are not using it anywhere else. I changed it in a8ee632, and I hope it reads better now 😃